.circleci/config.yml

```diff
@@ -234,28 +180,30 @@ jobs:
               --arg collectCoverage true \
               --attr doc
 
-      - save_cache:
-          name: "Cache Nix Store Paths"
+      - run:
+          name: "Push to Cachix"
           when: "always"
-          key: zkapauthorizer-nix-store-v4-{{ checksum "nixpkgs.rev" }}-ourdeps
-          paths:
-            - "/nix"
+          command: |
+            # Cribbed from
+            # https://circleci.com/blog/managing-secrets-when-you-have-pull-requests-from-outside-contributors/
+            if [ -n "$CIRCLE_PR_NUMBER" ]; then
+              # I'm sure you're thinking "CIRCLE_PR_NUMBER must just be the
+              # number of the PR being built".  Sorry, dear reader, you have
+              # guessed poorly.  It is also conditionally set based on whether
+              # this is a PR from a fork or not.
+              #
+              # https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables
+              echo "Skipping Cachix push for forked PR."
+            else
+              # https://docs.cachix.org/continuous-integration-setup/circleci.html
+              bash -c "comm -13 <(sort /tmp/store-path-pre-build | grep -v '\.drv$') <(nix path-info --all | grep -v '\.drv$' | sort) | cachix push $CACHIX_NAME"
```

Comment:
Sexy.

This uploads all the things, right? I am looking forward to see how it works out with the LRU caching and the 5 GB :)